VideoLocker/src/main/java/org/edx/mobile/view/CourseCombinedInfoFragment.java

```diff
@@ -341,19 +335,31 @@ private void populateAnnouncements(List<AnnouncementsModel> announcementsList) {
         if(announcementsList !=null && announcementsList.size()>0) {
             hideEmptyAnnouncementMessage();
 
-            final String divider = "<div style='height:2px; width:100%; background-color: #E2E3E5'></div>";
-
-            StringBuffer buff = new StringBuffer();
-
-            // add meta viewport
-            buff.append("<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">");
-
-            for (AnnouncementsModel m : announcementsList) {
-                String entry = String.format("%s <br/><br/> %s <br/> %s <br/><br/>", m.getDate(), divider, m.getContent());
-                buff.append(entry);
+            try{
+                String cssFileContent = FileUtil.loadTextFileFromAssets(getActivity(), "css/handouts-announcements.css");
+                StringBuffer buff = new StringBuffer();
+                // add meta viewport
+                buff.append("<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">");
+                // add css file
+                buff.append("<head>");
+                buff.append("<style>");
+                buff.append(cssFileContent);
```

Comment:
I think the try should be around this part instead of around the entire thing? That way if the css fails to load everything else is still fine.

So it would be something like: 
... 
buff.append("<head">);
try {
  String cssFileContent = FileUtil.loadTextFileFromAssets(getActivity(), "css/handouts-announcements.css");
  buff.append("<style>");
  buff.append(cssFileContent);
  buff.append("</style>");
}
catch ... {
   log("css file missing");
  ... handle exception here ...
}
```